util/dvsim/Testplan.py

```diff
@@ -198,7 +208,7 @@ def _parse_hjson(filename):
         sys.exit(1)
 
     @staticmethod
-    def _create_testplan_elements(kind, raw_dicts_list):
+    def _create_testplan_elements(kind, raw_dicts_list, tags=None):
```

Comment:
I think this is only used at a call-site that passes `tags`? In which case, I'd suggest dropping the default argument value.